SoloProjects/clc80/Dashboard/Dashboard/DashboardCard.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DashboardCard.swift
+//  Dashboard
+//
+//  Created by Claudia Maciel on 12/13/20.
+//
+
+import UIKit
+
+class DashboardCard: UIView {
+    //MARK: - Properties
+    var image1 = UIImageView()
+    var image2 = UIImageView()
+    let title = UILabel()
+    let subTitle = UILabel()
+    
+    override init(frame: CGRect) {
+        super .init(frame: frame)
```

Comment:
Watch out for little inconsistencies of spacing like this. It can feel frustrating that this matters, but I assure you there are plenty of technical reviewers out there who will notice this kind of thing.

```suggestion
        super.init(frame: frame)
```